hns/hns_test.go

```diff
@@ -99,18 +104,66 @@ var _ = Describe("HNS wrapper", func() {
 		})
 
 		Specify("getting a single HNS network works", func() {
-			net, err := GetHNSNetwork(testHnsID)
+			net, err := GetHNSNetwork(testHnsNetID)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(net).ToNot(BeNil())
-			Expect(net.Id).To(Equal(testHnsID))
+			Expect(net.Id).To(Equal(testHnsNetID))
 		})
 
 		Specify("getting a single HNS network by name works", func() {
 			net, err := GetHNSNetworkByName(testNetName)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(net).ToNot(BeNil())
-			Expect(net.Id).To(Equal(testHnsID))
+			Expect(net.Id).To(Equal(testHnsNetID))
+		})
+
+		Context("Given a docker container", func() {
+			testContainerName := "test_container"
+			containerID := ""
+			containerConfig := &container.Config{
+				Image: "microsoft/nanoserver",
+			}
+
+			BeforeEach(func() {
+				docker, err := dockerClient.NewEnvClient()
+				Expect(err).ToNot(HaveOccurred())
+				r, err := docker.ContainerCreate(context.Background(),
+					containerConfig, nil, nil, testContainerName)
+				Expect(err).ToNot(HaveOccurred())
+				containerID = r.ID
+			})
+
+			AfterEach(func() {
+				docker, err := dockerClient.NewEnvClient()
+				err = docker.ContainerRemove(context.Background(), containerID, types.ContainerRemoveOptions{
+					RemoveVolumes: true,
```

Comment:
Are there any volumes? I don't think there are created any.
If it's to be future-proof consider adding also RemoveLinks.